pkg/controller/collection/collection_controller.go

```diff
@@ -44,19 +47,54 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Create Collection predicate
+	c_pred := predicate.Funcs{
+		GenericFunc: func(e event.GenericEvent) bool {
+			return false
+		},
+		UpdateFunc: func(e event.UpdateEvent) bool {
+			// Ignore updates to CR status in which case metadata.Generation does not change
+			return e.MetaOld.GetGeneration() != e.MetaNew.GetGeneration()
```

Comment:
This is really powerful, and I didn't realize what it was doing until I searched the doc for `.metadata.generation`.  Basically this is what prevents the reconciler from running again when the operator updates the status of something.  Would you mind changing the comment so it says this more explicitly?  Something like "Returning true only when the metadata generation has changed, allows us to ignore events where only the object status has changed, since the generation is not incremented when only the status changes".  Feel free to ignore this comment unless you're in here to change other things.

Reply:
ok, will do

Reply:
comment updated in commit https://github.com/kabanero-io/kabanero-operator/pull/186/commits/d71fb952c78211209a2063be652e0ab60d62e2ac